packages/ontario-frontend-cli/commands/ontario-add-package/handleAddPackage.js

```diff
@@ -10,17 +11,25 @@ async function handleAddOntarioPackageCommand(cmd = {}) {
       case 'eslint':
```

Comment:
Can we store the resolved path and reuse it for efficiency and readability?

const resolvedPath = path.resolve(process.cwd());